include/hipSYCL/sycl/types.hpp

```diff
@@ -80,6 +80,7 @@ using hash_class = std::hash<T>;
 using exception_ptr_class = std::exception_ptr;
 
 
+using half = short;
```

Comment:
Would it make sense to set this as an alias for `detail::hp_float` that is defined below?

Reply:
mhmm Sure, :)